src/dialog/overflow-menu/overflow-menu.component.ts

```diff
@@ -40,13 +41,14 @@ import { OverflowMenuDirective } from "./overflow-menu.directive";
 			class="bx--overflow-menu"
 			[placement]="placement"
 			tabindex="0">
-			<svg focusable="false" class="bx--overflow-menu__icon" width="3" height="15" viewBox="0 0 3 15">
+			<svg *ngIf="!customIcon" focusable="false" class="bx--overflow-menu__icon" width="3" height="15" viewBox="0 0 3 15">
```

Comment:
Seems like `*ngIf="condition; else template` combo (like in accordion-item.component.ts for example) should work here.
That would make it clearer that it's a on-or-the-other situation here.

Reply:
Added 👍